server/src/main/java/org/elasticsearch/common/lease/Releasables.java

```diff
@@ -46,6 +47,15 @@ public static void close(Iterable<? extends Releasable> releasables) {
         close(releasables, false);
     }
 
+    /** Release the provided {@link Releasable}. */
+    public static void close(@Nullable Releasable releasable) {
```

Comment:
Same optimization already added to `IOUtils` for a single `Releasable` since we now use that heavily in closing the releasable stream.